test012.c:6: エラー: `A' は宣言されていません.
test012.c:6: エラー: `B' は宣言されていません.
test012.c:6: エラー: `A' の宣言で `int' が仮定されました.
test012.c:6: エラー: `A' が再宣言されています.
test012.c:6: エラー: ここで宣言されていました.
test012.c:6: エラー: `;' で構文エラーです.
test012.c:10: エラー: `b' は宣言されていません.
test012.c:10: エラー: `;' で構文エラーです.
test012.c: 函数 `f' :
test012.c:14: エラー: `C' は宣言されていません.
test012.c:14: エラー: `D' は宣言されていません.
test012.c:14: エラー: `;' で構文エラーです.
test012.c: 函数 `f' :
test012.c:18: エラー: `D' は宣言されていません.
test012.c:18: エラー: `d' は宣言されていません.
test012.c:18: エラー: `D' の宣言で `int' が仮定されました.
test012.c:18: エラー: `D' が再宣言されています.
test012.c:18: エラー: ここで宣言されていました.
test012.c:18: エラー: `;' で構文エラーです.
